notebooks/official/automl/sdk_automl_text_sentiment_analysis_online.ipynb

```diff
@@ -29,7 +29,7 @@
         "id": "title"
```

Comment:
<p>What is with the &lt;/span&gt; HTML tag?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>I didn't find &lt;/span&gt; HTML tag inside notebook. It might be something rendering issues.</p>

Reply:
didn't find </span> html tag.